blue_bringup/config/robot_parameters_right.yaml

```diff
@@ -21,8 +21,8 @@ blue_hardware:
     gear_ratios: [-7.1875, 7.1875, 8.2444852941, 7.1875, 8.24448529412, 7.1875, 8.24448529412, 14.4]
     joint_torque_directions: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
     id_torque_gains: [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0]
-    softstop_min_angles: [-3.3, -2.25, -2.6, -2.25, -2.6, -2.25, -2.6, -1.45]
-    softstop_max_angles: [2.2, 0.5, 2.6, 0.5, 2.6, 0.2, 2.6, 1.45]
+    softstop_min_angles: [-3.5, -2.45, -2.6, -2.45, -2.6, -2.45, -2.6, -1.65]
+    softstop_max_angles: [2.4, 0.6, 2.6, 0.6, 2.6, 0.6, 2.6, 1.65]
```

Comment:
Should these be updated in the left parameter yaml as well?

Reply:
yes will update that as well

Reply:
I think this is just a short term fix. What we should do is have the joint calibration look at the hard stops instead.